packages/dendron-cli/src/utils/build.ts

```diff
@@ -181,12 +181,37 @@ export class BuildUtils {
    * @param param0
    * @returns
    */
-  static packagePluginDependencies({ fast }: { fast?: boolean }) {
-    $(`yarn build:prod`, { cwd: this.getPluginRootPath() });
-    return $(`vsce package --yarn`, {
+  static async packagePluginDependencies({
+    fast,
+    quiet,
+  }: {
+    fast?: boolean;
+    quiet?: boolean;
+  }) {
+    const out = $$(`yarn build:prod`, {
       cwd: this.getPluginRootPath(),
       env: fast ? { SKIP_SENTRY: "true" } : {},
     });
+    if (!quiet) {
+      out.stdout?.pipe(process.stdout);
+    }
+    await out;
+    // TODO: not sure why this is generated but its not used by extension
+    const extensionBak = path.join(
+      this.getPluginRootPath(),
+      "dist",
+      "extension.jsbak"
+    );
+    if (fs.existsSync(extensionBak)) {
```

Comment:
Can you also add to `.vscodeignore`?

Reply:
actually, rather than delete this during build scripts, I think adding to `.vscodeignore` is a better fix.

Reply:
done